README.md

```diff
@@ -54,15 +117,20 @@ For consistency's sake, let's go ahead and rename `destination` back to `origin`
 git remote rename destination origin
 ```
 
-## `git push` + `git pull`
-
-Now that we added a remote repo, there are two actions. We can send our latest work to and retrieve the latest work from the remote.
+## Use `git push` to Send Code to the Remote Repo
 
-### `git push`
+Now that we added a remote repo, we need to send our latest work the remote.
 
-We use this command when we want to send some code from the local repository to the associated remote repository.
+We use this command when we want to send some code from the local repository to
+the associated remote repository.
 
-`git push` takes two arguments. The first is the name of the remote repo. Remember, `origin` is just an alias that refers to the repository name. You don't actually have to enter the repository name. Instead, you can just use `origin`. The second is the name of the remote branch you want to send code to. In the example below, we're pushing to the master branch of our remote repository, referred to as `origin`. To find all the branch names, run `git branch -r`.
+The `git push` command takes two arguments. The first is the name of the remote
+repo. Remember, `origin` is just an alias that refers to the repository name.
+You don't actually have to enter the repository name. Instead, you can just use
+`origin`. The second is the name of the remote branch you want to send code to.
+In the example below, we're pushing to the master branch of our remote
+repository, referred to as `origin`. To find all the remote branch names, run
+`git branch -r`.
```

Comment:
Not sure we need this yet.

We've not discussed branching.

Let's focus on the happy path.

Maybe we could talk about pushing local master to remote `test1` –– but that doesn't seem like a big win.